src/Samples.Shared/Managers/ApiKeyManager.cs

```diff
@@ -48,7 +48,7 @@ public static string ArcGISDeveloperApiKey
             {
                 // An Application programming interface key (API key) is a unique identifier used to authenticate a user, developer, or calling program with a server portal.
```

Comment:
I feel like this may be better... but up to you

An API key is a unique long-lived access token that is used to authenticate and monitor requests to ArcGIS location services and private portal items. You can create and manage an API key using your portal when you sign in with an
ArcGIS Location Platform account or an ArcGIS Online account with administrator access or a custom role that has the Generate API keys privilege. To learn how to create and manage API keys, go to the https://links.esri.com/create-an-api-key tutorial to create a new API key. You must ensure that your API key has the correct privileges to access secure resources.